### DISCUSSION TITLE
Benchmarking

### DISCUSSION THREAD

**Discussion by JoviDeCroock** [tw-in-js/twind#40] [General]
Currently the benchmarks seem geared towards testing the JS-execution and less of the CSSOM impact on the DOM, what would you think about moving to a more similar benchmark to [`styled-components`](https://github.com/styled-components/styled-components/tree/master/packages/benchmarks). The SC-benchmark is considered to be the most solid one at drawing out the actual perf impact.

The currently used one will only draw out the JS-impact of the styling while that concern is less prevalent in a DOM-world.

**Comment by lukejacksonn** [comment]
Ahh yes, I think @cristianbote mentioned this to me once before and it makes sense. It would be nice to know the end-to-end impact so to speak. I will try pulling in their test suite and then adding some Twind specific tests. I presume that is the approach you are suggesting we take?

**Reply by JoviDeCroock** [reply]
Yes, it's widely seen as one of the accurate ones since it's doing it in an actual browser and the bench itself is performing operations you'll see return in applications.

**Reply by sastan** [reply]
Sounds good.

**Reply by lukejacksonn** [reply]
Managed to get this setup today, here are some preliminary results:

![image](https://user-images.githubusercontent.com/1457604/103382134-bfc11380-4ae5-11eb-9cfd-e03faadaa1fb.png)
